view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -56,13 +56,112 @@ define(
                 return this;
             },
 
+            /**
+             * Get Installment minimum
+             * this function respects info from: https://www.omise.co/installment-payment
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @return {integer}
+             */
+            getInstallmentMinimum(id) {
+                switch (id) {
+                    case 'kbank':        return 500;
+                    case 'bbl':          return 500;
+                    case 'bay':          return 300;
+                    case 'first_choice': return 300;
+                    case 'ktc':          return 300;
+                    default:             return NaN;
```

Comment:
Not sure why defaulting to `NaN`? Won't this break stuff if a new instalment provider appears? Or are they hardcoded?

Reply:
no new installment provider until we hardcode one.

Reply:
And if they are hardcoded, this `default` is redundant?

Reply:
default is good practice.. that is how they taught me during studies..
it is that you have covered everything.

Reply:
but might delete that.. if omise has different rules.. @guzzilar

Reply:
```js
getInstallmentMinimum(id) {
   return {kbank:500, bbl:500}[id] || 300;
}
```
Would be another way to go

Reply:
mater of taste.. IMHO code is more readable if I show all installment terms of all brands.
they also might be changed and must come from capabilities API.